geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -11,32 +11,70 @@ use crate::{BoundingRect, TriangulateDelaunay};
 
 type Result<T> = result::Result<T, VoronoiDiagramError>;
 
+/// The default value for determining if the slope of a line is
+/// near zero or near infinite.
+///
+/// A near infinite slope is greater than this value.
+///
+/// A near zero slope is less than `1 / DEFAULT_SLOPE_THRESHOLD`.
 pub const DEFAULT_SLOPE_THRESHOLD: f64 = 1e3;
 
+/// The components of a Voronoi Diagram
 #[derive(Debug, Clone)]
 pub struct VoronoiComponents<T: GeoFloat> {
+    /// The Delaunay Triangles used to construct the Voronoi Diagram
     pub delaunay_triangles: Vec<Triangle<T>>,
+    /// The vertices of the Voronoi Diagram
     pub vertices: Vec<Coord<T>>,
+    /// The lines of the Voronoi Diagram
     pub lines: Vec<Line<T>>,
+    /// This is an index of Delaunay Triangles that are / have neighours
+    /// the first value is the index of the triangle and the second value
+    /// is the index of a neighbouring triangle.
+    /// If the triangle does not have a neighbour the value is None.
     pub neighbours: Vec<(Option<usize>, Option<usize>)>,
 }
 
+/// The clipping mask used to trim the infinity lines of
+/// the Voronoi Diagram.
 pub type ClippingMask<T> = Polygon<T>;
 
+/// Compute the Voronoi Diagram for a given set of points.
+/// This method uses the property that Delaunay Triangulation
```

Comment:
Triangulation -> triangulation
I searched the geo codebase and they dont seem to be capitalising this word except in Rust names.